prime-router/src/main/kotlin/azure/DatabaseAccess.kt

```diff
@@ -465,6 +472,11 @@ class DatabaseAccess(private val create: DSLContext) : Logging {
      */
 
     companion object {
+        /**
+         * Global var.  Set to false prior to the lazy init, to prevent flyway migrations
+         */
+        var isFlywayMigrationOK = true
```

Comment:
Please look at this carefully.   We had a dangerous bug where `./prime test --env staging`, run locally on a laptop, caused Flyway to push unwanted migrations onto staging.   This occurred because `./prime test` needs to access the staging database to check results.

My solution here is a global variable always set to true, but then set to false only by TestReportStream, prior to the lazy init of the database connectivity.